tidb-cloud/scale-tidb-cluster.md

```diff
@@ -11,22 +11,67 @@ aliases: ['/tidbcloud/beta/scale-tidb-cluter']
 > - Currently, you cannot scale a [Developer Tier cluster](/tidb-cloud/select-cluster-tier.md#developer-tier).
 > - When a cluster is in the scaling status, you cannot perform any new scaling operations on it.
 
-## Scale a cluster
+You can scale a TiDB cluster in the following dimensions:
 
-To scale a cluster, take the following steps:
+- Node number of TiDB, TiKV, and TiFlash
+- Storage size of TiKV and TiFlash
+- Node size (including vCPUs and memory) of TiDB, TiKV, and TiFlash
 
-1. Navigate to **Active Clusters** page and click the name of a cluster that you want to scale. The overview page of the cluster is displayed.
+For information about how to determine the size of your TiDB cluster, see [Determine Your TiDB Size](/tidb-cloud/size-your-cluster.md).
+
+## Change node number
+
```

Comment:
Please add a short description between the two headings.

Reply:
Thanks for the comment. The short description is added now.

Reply:
LGTM. Thanks~